net/modemmanager/files/usr/sbin/ModemManager-wrapper

```diff
@@ -28,6 +28,9 @@ main() {
 	mm_report_events_from_cache
 
 	wait "$CHILD"
+
+	# Set all configured interfaces as unavailable
+	mm_cleanup_interfaces
```

Comment:
There is another `mm_cleanup_interfaces` call just before launching the MM daemon. Was that call doing anything before these changes, given that it required `mmcli` to work?

Reply:
Since the MM daemon should be down at this point, the call likely did nothing before